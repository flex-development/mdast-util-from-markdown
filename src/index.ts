/**
 * @file Package Entry Point
 * @module mdast-util-from-markdown
 */

export type * from 'mdast'
export type {
  Encoding,
  Event,
  ParseOptions,
  Point,
  Token,
  TokenizeContext,
  Value
} from 'micromark-util-types'
export { default as compiler } from './compiler'
export { default as fromMarkdown } from './from-markdown'
export * from './handles'
export * as handles from './handles'
export type * from './types'
