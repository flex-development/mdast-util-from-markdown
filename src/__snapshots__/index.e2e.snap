// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:mdast-util-from-markdown > should expose public api 1`] = `
[
  "compiler",
  "enterdata",
  "enterlistitemvalue",
  "enterlistordered",
  "enterreference",
  "exitatxheadingsequence",
  "exitautolinkemail",
  "exitautolinkprotocol",
  "exitcharacterreference",
  "exitcharacterreferencemarker",
  "exitcharacterreferencevalue",
  "exitcodefenced",
  "exitcodefencedfence",
  "exitcodefencedfenceinfo",
  "exitcodefencedfencemeta",
  "exitcodeindented",
  "exitcodetext",
  "exitdata",
  "exitdefinitiondestinationstring",
  "exitdefinitionlabelstring",
  "exitdefinitiontitlestring",
  "exithardbreak",
  "exithtml",
  "exitimage",
  "exitlabel",
  "exitlabeltext",
  "exitlineending",
  "exitlink",
  "exitreferencestring",
  "exitresource",
  "exitresourcedestinationstring",
  "exitresourcetitlestring",
  "exitsetextheading",
  "exitsetextheadinglinesequence",
  "exitsetextheadingtext",
  "fromMarkdown",
  "handles",
]
`;
